cloud/amazon/ec2_elb_lb.py

```diff
@@ -708,6 +719,12 @@ def _set_connection_draining_timeout(self):
             attributes.connection_draining.enabled = False
             self.elb_conn.modify_lb_attribute(self.name, 'ConnectionDraining', attributes.connection_draining)
 
+    def _set_idle_timeout(self):
+        attributes = self.elb.get_attributes()
+        if self.idle_timeout is not None and self.idle_timeout != attributes.connecting_settings.idle_timeout:
```

Comment:
I was following the convention from within _set_connection_raining_timeout(self):
```python
 if self.connection_draining_timeout is not None:
```

What convention should I use for checking for defined ?  I could try something like:

```python
try:
    if self.idle_timeout != attributes.connecting_settings.idle_timeout:
        attributes.connecting_settings.idle_timeout = self.idle_timeout
        self.elb_conn.modify_lb_attribute(self.name, 'connectingsettings', attributes.connecting_settings)
except NameError: pass
```